### PR TITLE
refactoring footer classes and styles

### PR DESCRIPTION
**Pull request by rudenko-vl**
fixed class names by BEM, added vendor prefixes.

### CODE REVIEW

File: src/partials/layout/_footer.html [hunk 1]
```diff
@@ -20,51 +20,59 @@
         /></picture>
       </a>
     </div>
-    <div class="footer-container">
+    <div class="footer__container">
       <nav class="footer__nav">
-        <ul class="footer__nav-list list">
-          <li class="footer__nav-item">
-            <a class="footer__nav-link link" href="./index.html"
+        <ul class="footer__nav--list list">
```
Comment (Zolotarova-Nataliia) [inline src/partials/layout/_footer.html:25]: The element name is separated from the block name by two underscores (__).

Comment (Zolotarova-Nataliia) [reply src/partials/layout/_footer.html:25]: footer__nav must be block, not element.

File: src/partials/layout/_footer.html [hunk 2]
```diff
@@ -20,51 +20,59 @@
...
+          <li class="footer__nav--item">
+            <a class="footer__nav--link link" href="./index.html"
```
Comment (Zolotarova-Nataliia) [inline src/partials/layout/_footer.html:27]: all childs of nav must be elements of block footer-nav

**Review by Zolotarova-Nataliia — COMMENTED** [review]
You need to read BEM methodology documentation and change all names of classes.

File: src/partials/layout/_footer.html (hunk 1 above)
Comment (rudenko-vl) [reply src/partials/layout/_footer.html]: changed

File: src/partials/layout/_footer.html (hunk 2 above)
Comment (rudenko-vl) [reply src/partials/layout/_footer.html]: changed

File: src/partials/layout/_footer.html [hunk 3]
```diff
@@ -21,88 +21,78 @@
...
-                class="footer__contact-tel-link link"
-                href="tel:+380503454423"
-                >+38 (050) 345 44 23</a
-              >
-            </li>
-          </ul>
-        </address>
-      </div>
+      <address class="footer-contact">
+        <p class="footer-contact__mail--heading">Email</p>
```
Comment (Zolotarova-Nataliia) [inline src/partials/layout/_footer.html:72]: Rename footer-contact__mail--heading by BEM. BEM block: "block-name", BEM element: "block-name__element"

Comment (rudenko-vl) [reply src/partials/layout/_footer.html]: fixed

Comment (Androborn) [reply src/partials/layout/_footer.html]: Still not by BEM - corrections necessary

Comment (rudenko-vl) [reply src/partials/layout/_footer.html]: corrected

File: src/sass/layout/_footer.scss [hunk 4]
```diff
@@ -35,64 +57,115 @@
...
+.footer-social__item {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+.footer-social__item:not(:last-child) {
   margin-bottom: 8px;
 }
-.footer__social-link {
+.footer-social__link {
```
Comment (Androborn) [inline src/sass/layout/_footer.scss:127]: Looks like this class styles are doubled - see line 119.
Please either explain the doubling reason, or remove it.

File: src/sass/layout/_footer.scss [hunk 5]
```diff
@@ -35,64 +57,115 @@
...
+.footer-social__list {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
   flex-direction: column;
+}
+.footer-social__list {
```
Comment (Androborn) [inline src/sass/layout/_footer.scss:96]: Looks like this class styles are doubled - see line 87.
Please either explain the doubling reason, or remove it.

Comment (rudenko-vl) [reply src/sass/layout/_footer.scss]: deleted duplicate entry

File: src/sass/layout/_footer.scss (hunk 4 above)
Comment (rudenko-vl) [reply src/sass/layout/_footer.scss:118]: the footer-social__link class was repeated for the hover / focus. Made one entry with &.

**Comment by Androborn** [comment]
Causes too many conflicts - needs to be re-submitted, after pulling main.

**Pull request closed without merging**